server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const api = require('./server/routes/api');
+const path = require('path');
+
+const port = 2053;
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, 'dist')));
+app.use(express.static(path.join(__dirname, 'node_modules')));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use('/', api);
+
+mongoose.connect('mongodb://localhost/Weather', { useNewUrlParser: true });
```

Comment:
this line is usually going at the upper part of this file,  put it at line #6 or so, after declaring the variables